packages/ui/src/layout/ResizableGridLayout/index.tsx

```diff
@@ -341,6 +341,13 @@ const ResizableGridLayout = ({
         value: hidden === undefined ? true : !hidden,
     }));
 
+    // Save config if it's the first the use open the layout
```

Comment:
Moi et l'anglais hein mais je crois qu'il y a un the en trop
```suggestion
    // Save config if it's the first use open the layout
```